turbobt/substrate/pallets/author.py

```diff
@@ -10,14 +10,20 @@
 from .state import RuntimeVersion
 
 
+class Era(typing.TypedDict):
```

Comment:
why a typed dictionary and not a dataclass? using data classes allows you to avoid stuff like `if era and "current" not in era and "phase" not in era:`